omod/src/main/java/org/openmrs/module/spa/SpaActivator.java

```diff
@@ -14,16 +14,35 @@
 import org.openmrs.api.context.Context;
 import org.openmrs.module.BaseModuleActivator;
 
+import javax.servlet.ServletContext;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import static org.openmrs.module.spa.SpaConstants.BUNDLED_FRONTEND_DIRECTORY;
+import static org.openmrs.module.spa.SpaConstants.GP_LOCAL_DIRECTORY;
+
 @Slf4j
 public class SpaActivator extends BaseModuleActivator {
 
 	// here so we can register the listener on load and de-register it when stopped
 	private GlobalPropertyListener spaDirectoryResolver = null;
 
+	public static ServletContext servletContext;
```

Comment:
Don't we need to make this a Spring bean and make it servlet context aware for this to get a value?